model_info/ukwiki.wp10.md

```diff
@@ -0,0 +1,85 @@
+Model Information:
+	 - type: GradientBoosting
+	 - version: 0.8.0
+	 - params: {'presort': 'deprecated', 'max_leaf_nodes': None, 'multilabel': False, 'validation_fraction': 0.1, 'ccp_alpha': 0.0, 'min_impurity_decrease': 0.0, 'center': True, 'min_samples_leaf': 1, 'random_state': None, 'warm_start': False, 'init': None, 'loss': 'deviance', 'verbose': 0, 'scale': True, 'n_iter_no_change': None, 'tol': 0.0001, 'max_features': 'log2', 'max_depth': 7, 'subsample': 1.0, 'min_weight_fraction_leaf': 0.0, 'n_estimators': 500, 'population_rates': None, 'learning_rate': 0.01, 'criterion': 'friedman_mse', 'label_weights': None, 'min_impurity_split': None, 'labels': ['I', 'II', 'III', 'IV', 'ВС', 'ДС'], 'min_samples_split': 2}
+	Environment:
+	 - revscoring_version: '2.8.2'
+	 - platform: 'Linux-4.9.0-8-amd64-x86_64-with-debian-9.4'
+	 - machine: 'x86_64'
+	 - version: '#1 SMP Debian 4.9.144-3.1 (2019-02-19)'
+	 - system: 'Linux'
+	 - processor: ''
+	 - python_build: ('default', 'Sep 27 2018 17:25:39')
+	 - python_compiler: 'GCC 6.3.0 20170516'
+	 - python_branch: ''
+	 - python_implementation: 'CPython'
+	 - python_revision: ''
+	 - python_version: '3.5.3'
+	 - release: '4.9.0-8-amd64'
+	
+	Statistics:
+	counts (n=899):
+		label      n         ~I    ~II    ~III    ~IV    ~ВС    ~ДС
+		-------  ---  ---  ----  -----  ------  -----  -----  -----
+		'I'      150  -->    57     34      13      6     18     22
+		'II'     149  -->    30     63      23     21      5      7
+		'III'    150  -->     9     41      51     44      2      3
+		'IV'     150  -->     6     19      42     83      0      0
+		'ВС'     150  -->    13     12       2      0     86     37
+		'ДС'     150  -->    39     11       3      0     39     58
+	rates:
+		              'I'    'II'    'III'    'IV'    'ВС'    'ДС'
```

Comment:
By changing the order in the cv_train call, it'll make this order appear correctly.